spec/models/password_strategies_spec.rb

```diff
@@ -3,6 +3,8 @@
 describe Clearance::User do
   subject do
     class UniquenessValidator < ActiveModel::Validator
+      undef validate
```

Comment:
I couldnt really figure this out. I think it may have to do with this being a "subject" which gets continually re-evaluated for each spec.